apps/condo/domains/ticket/utils/serverSchema/exportTicketBlanksToPdf.js

```diff
@@ -301,35 +304,48 @@ const generatePdf = (replaces) => {
                       <!-- NOTE This text does not wrap, only one line -->
                       <text x="0" y="10" class="big">${i18n.workCompleted}</text>
                     
-                      <rect x="0" y="30" width="144" height="1" fill="#222222"/>
+                      <rect x="0" y="30" width="118" height="1" fill="#222222"/>
                       <text x="0" y="40" class="small">${i18n.executorFullName}</text>
                       
-                      <rect x="161" y="30" width="125" height="1" fill="#222222"/>
-                      <text x="161" y="40" class="small">${i18n.executorSignature}</text>
+                      <rect x="131" y="30" width="118" height="1" fill="#222222"/>
+                      <text x="131" y="40" class="small">${i18n.executorSignature}</text>
                       
-                      <rect x="0" y="60" width="144" height="1" fill="#222222"/>
+                      <rect x="0" y="60" width="118" height="1" fill="#222222"/>
                       <text x="0" y="70" class="small">${i18n.clientFullName}</text>
                       
-                      <rect x="161" y="60" width="125" height="1" fill="#222222"/>
-                      <text x="161" y="70" class="small">${i18n.clientSignature}</text>
+                      <rect x="131" y="60" width="118" height="1" fill="#222222"/>
```

Comment:
Looks like copy-paste

Reply:
Yes.. But it doesn't seem to make sense to make new logic for this, since it's a single and constant element 

It will look harder